Wrapping/Python/Examples/use_case_02.py

```diff
@@ -116,7 +123,8 @@ def import_csv_xyz_vertices_1(file_path: str):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('--input_file')
+    parser.add_argument('--output_file')
```

Comment:
This should be 'output_dir' and any output file that is written (.csv and dream3d files), those names should be prefixed with the output_dir path that is passed into the program